diffrax/_term.py

```diff
@@ -788,6 +788,16 @@ def _to_vjp(_y, _diff_args, _diff_term):
         return dy, da_y, da_diff_args, da_diff_term
 
 
+class LinearODETerm(ODETerm):
+    operator: lx.MatrixLinearOperator | lx.DiagonalLinearOperator
+
+    def __init__(self, operator: lx.MatrixLinearOperator | lx.DiagonalLinearOperator):
+        self.operator = operator
+        vf = lambda t, y, args: self.operator.mv(y)
+        super().__init__(vector_field=vf)
```

Comment:
Btw, I try to avoid subclassing concrete classes -- c.f. https://docs.kidger.site/equinox/pattern/

Also, does this strictly need to be an ODE term? I think in other problems, we might be interested in expressing linear terms in this way (for any `operator: lx.AbstractLinearOperator`) regardless of the control.